### PR TITLE
#460: Remove Teradata references

### LINKED ISSUE #460: Remove Teradata references
# Acceptance Criteria

* Teradata references are removed.

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #460.

### CODE REVIEW

File: doc/changes/changelog.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changes
 
-* [6.0.0](changes_6.0.0.md)
+* [6.0.0](changes_6.0.1.md)
```
Comment (chiaradiamarcelo) [inline doc/changes/changelog.md:3]:
```suggestion
* [6.0.1](changes_6.0.1.md)
```

File: doc/changes/changes_6.0.1.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Exasol Virtual Schemas 6.0.0, released 2021-03-??
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_6.0.1.md:1]: Are we going to release this? right now this project contains only doc.

Comment (chiaradiamarcelo) [inline doc/changes/changes_6.0.1.md:1]:
```suggestion
# Exasol Virtual Schemas 6.0.1, released 2021-03-??
```

File: doc/changes/changelog.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Changes
 
-* [6.0.0](changes_6.0.0.md)
```
Comment (chiaradiamarcelo) [inline doc/changes/changelog.md:3]: why is this line removed?

File: doc/changes/changelog.md (hunk 1 above)
Comment (redcatbear) [reply doc/changes/changelog.md:3]: Fixed.

File: doc/changes/changes_6.0.1.md (hunk 2 above)
Comment (redcatbear) [reply doc/changes/changes_6.0.1.md:1]: Fixed.

File: doc/changes/changelog.md (hunk 3 above)
Comment (redcatbear) [reply doc/changes/changelog.md:3]: Typo. Fixed.

**Review by chiaradiamarcelo — APPROVED** [review]
